cluster/gce/gci/configure-helper.sh

```diff
@@ -609,7 +616,7 @@ function prepare-etcd-manifest {
   local etcd_cluster=""
   local cluster_state="new"
   for host in $(echo "${INITIAL_ETCD_CLUSTER:-${host_name}}" | tr "," "\n"); do
-    etcd_host="etcd-${host}=http://${host}:$3"
+    etcd_host="etcd-${host}=https://${host}:$3"
```

Comment:
this looks like it sets https for every setup, regardless of HA or non-HA. should we leave explicit non-HA using localhost http, so that we don't have to have the apiserver connect insecurely to etcd?